test/core/iomgr/combiner_test.cc

```diff
@@ -24,14 +24,20 @@
 #include <grpc/support/alloc.h>
 #include <grpc/support/log.h>
 
+#include "src/core/lib/event_engine/default_event_engine_factory.h"
 #include "src/core/lib/gpr/useful.h"
 #include "src/core/lib/gprpp/thd.h"
 #include "test/core/util/test_config.h"
 
 TEST(CombinerTest, TestNoOp) {
   gpr_log(GPR_DEBUG, "test_no_op");
   grpc_core::ExecCtx exec_ctx;
-  GRPC_COMBINER_UNREF(grpc_combiner_create(), "test_no_op");
+  GRPC_COMBINER_UNREF(
+      grpc_combiner_create(
+          std::shared_ptr<grpc_event_engine::experimental::EventEngine>(
+              grpc_event_engine::experimental::DefaultEventEngineFactory()
```

Comment:
Small suggestion: I recommend using `CreateEventEngine` to prefer the common/public path in tests, rather than internal APIs. That would also future-proofs this against test rig changes that that exercise tests with different factories.

Reply:
done